draft-wang-ppm-differential-privacy.md

```diff
@@ -445,13 +458,20 @@ be trusted to execute the protocol correctly (i.e., which parties are not
 corrupted by the attacker). We consider three, increasingly pessimistic trust
 models.
 
+> KT: Here we seem to be assuming corrupted = malicious. Is there any benefit to
```

Comment:
@kunal-talwar how to distinguish between malicious and honest-but-curious parties?

Reply:
I suspect the distinction won't be helpful. For the purposes of (differential) privacy, suppose that all parties corrupted by the attacker are fully malicious and not just semi-honest.

Reply:
Putting it into an issue instead: #28